docs/installation-and-operations/configuration/incoming-emails/README.md

```diff
@@ -184,17 +184,18 @@ The subject of the work package that shall be created is derived from the subjec
 
 Other available keys for the email are:
 
-|Key|Description|Example|
-|---|---|---|
-| Project | sets the project. Use the project identifier | Project:test\_project |
-| Assignee | sets the assignee. Use the email or login of the user | Assignee:test.nutzer@example.org |
-| Type | sets the type | type:Milestone |
-| Version | sets the version | version:v4.1.0 |
-| Start date | sets the start date | start date:2015-02-28 |
-| Due date | sets the finish date |  |
-| Done ratio | sets the done ratio. Use a number | Done ratio:40 |
-| Status | sets the status | Status:closed |
-| priority | sets the priority | priority:High |
+| Key             | Description                                           | Example                          |
+|-----------------|-------------------------------------------------------|----------------------------------|
+| Project         | sets the project. Use the project identifier          | Project:test\_project            |
+| Assignee        | sets the assignee. Use the email or login of the user | Assignee:test.nutzer@example.org |
+| Type            | sets the type                                         | type:Milestone                   |
+| Version         | sets the version                                      | version:v4.1.0                   |
+| Start date      | sets the start date                                   | start date:2015-02-28            |
+| Due date        | sets the finish date                                  |                                  |
+| Estimated hours | sets the estimated hours. Use a number                | Estimated hours:10.5             |
+| Remaining hours | sets the remaining hours. Use a number                | Remaining hours:2.5              |
```

Comment:
That's where I do not know if we should use the old names or the new names (and have the old ones still working but undocumented). I suppose using the old names does not hurt, and it can still be revisited later.

Reply:
And the table looks nice now that you have added some padding to it.

Reply:
I second guessed on the names too but I thought it could be something to address when we ensure the same terms are used throughout the code base (solely work, remaining_work, percent_complete)